securetea.conf

```diff
@@ -5,5 +5,9 @@
 		"access_token": "XXXX",
 		"access_token_secret": "XXXX"
 	},
-	"debug": false
-}
\ No newline at end of file
+	"telegram": {
+		"token": "XXXX",
+		"user_id": "XXXX"
+	},
+	"debug": true
```

Comment:
I think default debug need to set to false. Do you have any specific requirement to set it true?

Reply:
No, no specific requirements, I forgot to update, sorry